kvault/src/iosMain/kotlin/com/liftric/kvault/KVault.kt

```diff
@@ -23,33 +23,63 @@ actual class KVault() {
     // SET OPERATIONS
     // ===============
 
+    /**
+     * Saves a string value in the keychain.
```

Comment:
Capitalise "keychain" since this is the name of the Framework.
```suggestion
     * Saves a string value in the Keychain.
```